wbooks/src/app/redux/index.js

```diff
@@ -0,0 +1,225 @@
+import * as AuthService from '../../services/authService';
```

Comment:
We divide redux into two files, `actions.js` and `reducers.js. Then, this is subdivided by responsability.
For example.
```
Redux
   - Authentication
            actions.js
            reducers.js
   -  Books
            actions.js
             reducers.js
 ```